### DISCUSSION TITLE
Hi, does git extension provide a icon overlay on windows explorer?

### DISCUSSION THREAD

**Discussion by TurnOffNOD** [gitextensions/gitextensions#10182] [Q&A]
### Feature description

Hi, when I was using tortoisegit, there is a git icon overlay. And hg on widows also has an icon overlay.
But I can't find when I am using gitextension. Does git extension provide a icon overlay on windows explorer?

### Environment

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version:

**Comment by RussKie** [comment]
No. TortoiseGit is a shell extension whereas Git Extensions is a fully-fledged git UI.

Could you describe a scenario when such thing could  be useful?
